source/reference/api/start.txt

```diff
@@ -76,6 +96,26 @@ Request
 Response
 ~~~~~~~~
 
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Example 2 - Start a Reversible Sync Job
+---------------------------------------
+
+The following example starts a synchronization job where ``cluster0`` is
+the source and ``cluster1`` is the destination, and specifies the
+required fields to allow the sync to be reversed.
```

Comment:
Consider:

is the destination. The X, Y, Z fields are permit the sync to be reversed.